auth-web/src/stores/business.ts

```diff
@@ -209,6 +209,21 @@ export const useBusinessStore = defineStore('business', () => {
     const entityResponse: AffiliationResponse[] = await OrgService.getAffiliatedEntities(currentOrganization.value.id)
     let affiliatedEntities: Business[] = []
 
+    // do a lear search on firms to fetch the latest names (alternate names array)
+    // set that array to firm entries in the entityResponse array
+    for (const response of entityResponse) {
+      if ([CorpTypes.SOLE_PROP, CorpTypes.PARTNERSHIP].includes(response.legalType) &&
+        (response.state === EntityStates.ACTIVE || response.state === EntityStates.HISTORICAL)) {
+        await BusinessService.searchBusiness(response.identifier).then(result => {
+          response.legalName = result.data.business.legalName
+          response.alternateNames = result.data.business.alternateNames
+        }).catch(error => {
+          console.log(error)
+        })
+      }
+    }
```

Comment:
Future improvement: build an array of calls to search and then run them concurrently. (Promise.all() or similar.)

Reply:
Travis left a better comment above. Resolving this one.